### PR TITLE
chore: linting and travis

### PR DESCRIPTION
**Pull request by hacdias**
I just @fabiofcferreira's Insta story and I couldn't contain myself not to do this PR. Adds linting and build on Travis.

I solved *some* linting issues, but you've got to solve the other ones. There are lots of them. Also added some comments where some stuff could be simplified.

That's it. Good work!

License: MIT
Signed-off-by: Henrique Dias <hacdias@gmail.com>

### CODE REVIEW

File: src/components/footer.js [hunk 1]
```diff
@@ -1,16 +1,26 @@
-import React, {Component} from 'react';
+import React, {Component} from 'react'
 
+// Here you could just use a functional component for better performance:
```
Comment (hacdias) [inline src/components/footer.js]: Stateless components should be functions.

**Comment by fabiofcferreira** [comment]
We've not been paying enough attention to the linting errors and to be honest, I thought Travis wouldn't help that much since we need the mvp ready asap. But right now, I think it makes sense.

This is literally the best surprise in the repository's lifetime ❤️

**Comment by hacdias** [comment]
Wanting a MVP ASAP shouldn't decay the code's quality.

**Comment by ulissesferreira** [comment]
Thanks for the work Henrique 😍. Funny because yesterday I was taking a look at my Github Student Pack and though about integrating Travis with connect 🤣

**Pull request merged**